src/main/scala/treadle/executable/ExecutionEngine.scala

```diff
@@ -200,6 +204,15 @@ class ExecutionEngine(
     val symbol = symbolTable(name)
 
     inputsChanged = true
+    if(symbolsPokedSinceEvaluation.contains(symbol)) {
+      if(verbose) {
+        println(s"updating circuit on second update of same input without clock advance")
```

Comment:
The default is to silently accept multiple updates, correct?

Reply:
Yes, this is an edge case that appears in testers-2 where richard is not calling step but instead just toggling an ordinary bool that is cast to a clock inside a module. I'm not that happy with the idiom, but it makes some sense, and we are bit straight-jacketed by the existing backend APIs